### ISSUE TITLE
Unknown attributes temperatureHvBatteryMin_K, temperatureHvBatteryMax_K

### ISSUE THREAD

**Issue by madd0** [tillsteinbach/WeConnect-python#158]
**Describe your car model and the situation in which the error was shown**
ID.3
Updates in [volkswagen_we_connect_id](https://github.com/mitch-dc/volkswagen_we_connect_id), which is using v0.58.0 reporting warnings of two new attributes.

**Paste the error without confidential information such as the VIN**

```
/vehicles/WVXXXX/domains/measurements/temperatureBatteryStatus: Unknown attribute temperatureHvBatteryMin_K with value 289.65
/vehicles/WVXXXX/domains/measurements/temperatureBatteryStatus: Unknown attribute temperatureHvBatteryMax_K with value 290.15
```

**Additional context**
N/A

**Comment by madd0** [comment]
My bad, the component's version is pinned to 0.55.1, but I believe the attributes are still unknown in the latest version.

**Comment by tillsteinbach** [comment]
That is nice! What version is the ID.3 running? I cannot see that value with a 3.0 car.

**Comment by tillsteinbach** [comment]
Released in v0.59.2

**Comment by madd0** [comment]
> That is nice! What version is the ID.3 running? I cannot see that value with a 3.0 car.

It's running 3.2
